lib/absinthe/language/input_value_definition.ex

```diff
@@ -25,7 +25,7 @@ defmodule Absinthe.Language.InputValueDefinition do
   defimpl Blueprint.Draft do
     def convert(node, doc) do
       %Blueprint.Schema.InputValueDefinition{
-        name: node.name,
+        name: Macro.underscore(node.name),
```

Comment:
The only names that stay canonically camelcase are type names. argument and field names get underscored.